cmd/statusd/main.go

```diff
@@ -106,6 +109,13 @@ func main() {
 		go startCollectingStats(interruptCh, backend.NodeManager())
 	}
 
+	// Sync blockchain and stop.
+	if *syncAndExit >= 0 {
+		if err = syncAndStopNode(backend.NodeManager()); err != nil {
+			log.Fatalf("Node synchronization failed: %v", err)
+		}
```

Comment:
there should be a `return`, right? :)

Reply:
`log.Fatalf` calls `os.Exit(1)` but I would use `err :=` as there is no point in overriding previous `err` value.

Reply:
yeah, but if there is _no_ error?

Reply:
@mandrigin Thanks for pointing out. :)

I think it should return if there is no error.

Reply:
@adambabik 
> but I would use `err :=` ...

I can reflect this change in the next push.